src/Console/ConfigurationResolver.php

```diff
@@ -401,52 +403,69 @@ private function resolveConfigPath()
             return;
         }
 
-        if (null === $this->path) {
+        if (empty($this->path)) {
             return;
         }
 
-        $path = realpath($this->path);
+        $paths = array_filter(array_map(
+            function ($path) {
+                return realpath($path);
+            },
+            $this->path
+        ));
 
-        if (false === $path) {
+        if (empty($paths)) {
```

Comment:
This is no longer possible I think?
0 can only be with `stdIn` which is covered at line 400?